lib/Cleantalk/ApbctWP/State.php

```diff
@@ -113,6 +113,8 @@ class State
         'data__protect_logged_in'                  => 1,
         // Do anti-spam tests to for logged in users.
         'data__use_ajax'                           => 1,
+        'data__use_ajax__type'                     => 1,
+        // Alternative cookies handler type: REST API - 0 / custom AJAX - 1 / WP AJAX - 2
```

Comment:
Please, update the commentary

Reply:
Fixed https://github.com/CleanTalk/wordpress-antispam/pull/42/commits/81711b89a9e34daa18bab356f75f1f126fa9023f